index.html

```diff
@@ -617,28 +617,51 @@ <h2>Add Ons</h2>
     <section class="container">
       <h1>Perks in Detail</h1>
       <div class="perks-row">
-        <div class="perks-col pic"></div>
-        <div class="perks-col">
-          WOW
+        <div class="perks-col pic">
+          <img src="./images/careers.jpg">
+        </div>
+        <div class="perks-col text">
+          <h1>Careers Fair and Careers Roundup</h1>
```

Comment:
can these be `h2`s to keep the hierarchy

Reply:
Yeah good call! Will do 🤙